python/src/pyraphtory/algorithms/sample.py

```diff
@@ -39,35 +38,39 @@ def parse(graph, tuple: str):
             graph.load(Source(lotr_spout, GraphBuilder(parse)))
 
             df = (graph
-                  .select(lambda vertex: Row(vertex.name(), vertex.degree()))
-                  .to_df(["name", "degree"]))
+                  .step(lambda vertex: vertex.set_state("name",vertex.name()))\
```

Comment:
Just as a note incase we forget, I believe Lucas was mentioning certain phrases mapping to function calls like degree. Definitely not for now, but for it we were to come back here